src/main/java/com/google/api/codegen/config/DiscoveryMethodModel.java

```diff
@@ -265,26 +264,23 @@ public boolean hasReturnValue() {
       return outputFields;
     }
 
-    List<FieldModel> outputField;
+    ImmutableList.Builder outputField = new Builder();
```

Comment:
Don't you want to parameterize this with `<FieldModel>`? (You should be getting a warning in your IDE)

Reply:
Done.